src/amo/components/SearchPage/index.js

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import Search from 'amo/components/Search';
+import { convertQueryParamsToFilters } from 'core/searchUtils';
+
+
+export function mapLocationToProps(state, ownProps) {
```

Comment:
Ah, fair enough especially for grep-ing. This function's intent is more to grab location info but that's a good point. 👍